docs/subtensor-nodes/subtensor-storage-query-examples.md

```diff
@@ -1136,7 +1136,7 @@ title: "Subtensor Storage Query Examples"
 
     substrate = SubstrateInterface(url="ws://127.0.0.1:9945")
     hotkey = Keypair.create_from_uri('//Alice').ss58_address
-    result = substrate.query('SubtensorModule', 'PendingdHotkeyEmission', [hotkey])
+    result = substrate.query('SubtensorModule', 'PendingHotkeyEmission', [hotkey])
```

Comment:
Similarly for this. The function exists with the typo on the chain, as  subtensorModule.pendingdHotkeyEmission: u64

Please revert this change.